rke/k8s_rke_system_images.go

```diff
@@ -5827,6 +5827,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled with v2.7.0
+		"v1.19.16-rancher2-1": {
```

Comment:
Note to reviewers: `v1.19.16-rancher2-1` will only show up for v2.7, it has already been released for v2.6 https://github.com/rancher/kontainer-driver-metadata/blob/release-v2.6/rke/k8s_rke_system_images.go#L5831